frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/LitigantHomeConfig.js

```diff
@@ -102,7 +102,7 @@ export const TabLitigantSearchConfig = {
   showTab: true,
   TabSearchConfig: [
     {
-      label: "All",
+      label: "Ongoing",
```

Comment:
Is it possible to add localization strings here @shambhavigoel1002 , as part of this PR or would this require some larger changes?